Apps/Medication/src/Delegates/RestMedStatementDelegate.cs

```diff
@@ -194,7 +194,7 @@ public bool ValidateProtectiveWord(string phn, string? protectiveWord, string hd
         {
             using (Source.StartActivity("ValidateProtectiveWord"))
             {
-                bool retVal = false;
+                bool isValid = false;
```

Comment:
NIT: Why are we changing server side code at all and why change this? I don’t think it has added any clarity since it’s a return value which is documented.

Reply:
I was trying to figure out where the protected word magic happens and thought the variable rename would make it much easier to understand this method. Not deadset on it, if retval is more clear that we can keep it :)

Reply:
It's just a NIT, I'm ok with the commit.

Reply:
isValid may get lost in a complex function isValid can be just a temporary variable. I'm ok with retVal as in this function it makes us think of the name of function which returns the state of validation of Protective Word.